executor/testdata/point_get_suite_out.json

```diff
@@ -59,9 +59,9 @@
       {
         "SQL": "select * from t2 where a in ('111','222') union all select a,c from t1 where (a,b) in ((1,1),(2,2))",
         "Plan": [
-          "Union_10 8020.00 root  ",
+          "Union_10 22.00 root  ",
```

Comment:
After `DeriveStats`, `DataSource(t2)`'s est. row count is 20, so Union's row count is 22. Then in the `findBestTask`, `DataSource(t2)` becomes `BatchPointGet` and it uses ranges to adjust the est. row count to 2. But `Union`'s row count won't change here.

Reply:
Why is the estimated row count of t2 is 20 in `DeriveStats()`? If it is not as expected, you can open a new issue to track it.

Reply:
First, because we didn't analyze the tables in this unit test, the tests are using pseudo statistics to estimate row count. It's basically using a pseudo row count 10000 to multiply selectivity 0.002 and make row count of t2 20. So I added analyze to this unit test.

Second, there's a bug that would cause common handle primary key can't be used to calculate selectivity. Then an inaccurate statistics will be used here, resulting in an inaccurate row count estimation. I created an issue for that. #19313